tfchain_bridge/pkg/bridge/mint.go

```diff
@@ -0,0 +1,160 @@
+package bridge
+
+import (
+	"context"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	hProtocol "github.com/stellar/go/protocols/horizon"
+	"github.com/threefoldtech/substrate-client"
+)
+
+// mint handler for stellar
+func (bridge *Bridge) mint(senders map[string]*big.Int, tx hProtocol.Transaction) error {
+	log.Info().Msg("calling mint now")
+
+	if len(senders) > 1 {
+		log.Info().Msgf("cannot process mint transaction, multiple senders found, refunding now")
+		for sender, depositAmount := range senders {
+			return bridge.refund(context.Background(), sender, depositAmount.Int64(), tx)
+		}
+	}
+
```

Comment:
Missing check senders > 0

Reply:
Why do we need this?

Reply:
Because it's a map and therefore can be empty

Reply:
like this? https://github.com/threefoldtech/tfchain_tft_bridge/blob/9b2e70aef062b41370291ffcdb6366f14070d90d/tfchain_bridge/pkg/bridge/mint.go#L31